R/create_fig_1.R

```diff
@@ -7,39 +7,31 @@
 create_fig_1 <- function(
   project_folder_name
 ) {
+  check_project_folder_name(project_folder_name)
 
-  par_setting <- nltt <- gen_model <- NULL
+  # ISSUE #138 STUB
 
-  df0 <- collect_nltt_stats(project_folder_name) # nolint internal function
-  df0$par_setting <- interaction(
-    df0$lambda,
-    df0$mu,
-    df0$nu,
-    df0$q,
-    df0$site_model,
-    df0$clock_model
-  )
+  lambda <- 0.2
+  mu <- 0.1
+  nu <- 2
+  q <- 0.1
 
-  df <- tidyr::gather(
-    df0,
-    "i",
-    "nltt",
-    (1:ncol(df0))[grepl("nltt", names(df0))]
+  df <- data.frame(
+   par_setting = c(1, 2),
+   nltt = c(0.1, 0.2),
+   gen_model = c("1", "2")
   )
-  xlabels <- unique(paste0(
-    df0$lambda,
-    "\n",
-    df$mu,
-    "\n",
-    df$nu,
-    "\n",
-    df$q,
-    "\n",
-    df$site_model,
-    "\n",
-    df$clock_model
-  ))
-
+  if (1) {
+    # need to call these variables to avoid notes in check
```

Comment:
This is a problem indeed. This is how I learned to solve it:

```
marg_log_lik <- NULL; rm(marg_log_lik) # nolint, fixes warning: no visible binding for global variable
```

Reply:
It doesn't really matter as it is just a temporary solution while we figure out how to produce figures. I wanna do it but I need a bit more information...

Reply:
This is even the recommended solution :sunglasses:

You may wonder why we need to do this in R, but it's because of a hard problem within R... :rainbow: